content/_data/changelogs/weekly.yml

```diff
@@ -2708,6 +2708,58 @@
       message: >
         Do not remove workspaces for projects with builds in progress.
     # pull: 3436 too minor (queue API messages)
+- version: "2.126"
+  date: 2018-06-03
+  changes:
+    - type: rfe
+      pull: 3471
+      issue: 50217
+      message: >
+        Robustness: A buggy <code>ComputerListener#onConfigurationChange</code> implementation should not block Jenkins startup.
+    - type: rfe
+      pull: 3447
+      issue: 51355
+      message: >
+        Diagnostics: Log stack traces in JEP-200 rejection messages when <code>jenkins.security.ClassFilterImpl</code> logging level is <code>FINE</code> or above.
+    - type: bug
+      issue: 51650
+      pull: 3466
+      message: >
+        Copying <em>Run</em> parameters did not work as expected as <code>RunParameterDefinition#copyWithDefaultValue</code> called the wrong constructor.
+    - type: bug
+      pull: 3462
+      issue: 51584
+      message: >
+        Actions created from a <code>TransientActionFactory</code> that got attached to an item in the queue are no longer persisted, which could previously lead to duplicate actions shown for builds.
+    - type: bug
+      pull: 3453
+      issue: 20272
+      message: >
+        Do not attempt to disconnect offline computers for not responding.
+    - type: bug
+      pull: 3454
+      issue: 51483
+      message: >
+        Restore implied dependency on JDK Tool Plugin from Apache HttpComponents Client 4 API Plugin to fix dependency problems.
+    - type: bug
+      pull: 3467
+      issue: 14632
+      message: >
+        Fix behaviour of <kbd>Advanced</kbd> button when a <code>section</code> element is nested inside.
```

Comment:
`kbd`? Why not just `em` or so?

Reply:
Because I misremembered the definition of `kbd`.